Misc/NEWS.d/next/Documentation/2021-06-23-15-21-36.bpo-39452.o_I-6d.rst

```diff
@@ -0,0 +1,3 @@
+Rewrote ``Doc/library/__main__.rst``. Broadened scope of the document to
+explicitly discuss and differentiate between ``__main__.py`` in packages
+versus the ``__name__ == '__main__'`` statement.
```

Comment:
Ditto about statement.

Reply:
```suggestion
versus the ``if __name__ == '__main__':`` statement.
```

Reply:
I've applied the same fix as before: statement => expression